pages/events/index.md

```diff
@@ -4,3 +4,82 @@ layout: events
 collection: events
 permalink: /events/
 ---
+
+<section id="conference" class="taxonomy__section">
+    <h2>US-RSE Conference</h2>
+    <hr style="width:40%" />
+
+<div class="row">
+
+    <div class="col" style="margin-bottom: 20px;">
+<div class="card h-100" id="usrse24">
+  <a href="{{ site.baseurl }}/usrse24/" class="stretched-link">
+  </a>
+  
+  {% assign color = "#790c63" %}
+
+  <div class="card-header" style="background-color: {{ color }};">
+      <div class="col-sm-8">
+        <h5 style="margin: 0px -15px; color: rgb(255, 255, 255)">
+          US-RSE'24: Albuquerque, NM
+        </h5>
+      </div>
+      <div class="col-xs-4 float-right">
+        <h6 style="text-align: right; color: rgb(168, 168, 168)">
+          October 15-17, 2024
+        </h6>
+      </div>
+  </div>
+  <div class="card-body">
+    <h6 class="card-subtitle mb-2">
+    Yesterday, Today, Tomorrow
+    </h6>
+    
+    <p class="card-text">
+      A celebration of all that RSEs have done for computing in the past, in the present, and in the future. 
+    </p>
+
+  </div>
+</div>
+</div>
+
+<div class="col" style="margin-bottom: 20px;">
+
+<div class="card h-100" id="usrse23">
+  <a href="{{ site.baseurl }}/usrse23/" class="stretched-link">
+  </a>
+  
+  {% assign color = "#717171" %}
+
+  <div class="card-header" style="background-color: {{ color }};">
```

Comment:
I think we should keep the events page just upcoming (and it's already listed on the archive). I believe this is the block to remove, but don't want to suggest it and get it wrong.